src/source_clang.cc

```diff
@@ -874,8 +874,8 @@ void Source::ClangViewAutocomplete::async_delete() {
   parse_thread_stop=true;
   delete_thread=std::thread([this](){
     //TODO: Is it possible to stop the clang-process in progress?
-    if(restart_parse_thread.joinable())
-      restart_parse_thread.join();
+    if(full_reparse_thread.joinable())
```

Comment:
Sadly, there is no way to stop a clang-parse/reparse or autocomplete once it has been started (at least I have not found it yet). Normally, this would be done by for instance setting some stop_process atomic variable to true, but libclang has no such method.